src/MessagingAttachment/MessagingAttachment.tsx

```diff
@@ -131,6 +131,9 @@ export type AttachmentFileType =
   | "txt"
   | null;
 
+// the set of fileTypes we are previewing in AttachmentPreviews
+export const previewableFileTypes = new Set(["jpg", "jpeg", "gif", "png", "svg"]);
```

Comment:
This should probably be capitalized to `PreviewableFileTypes` or `PREVIEWABLE_FILE_TYPES`

Reply:
I just noticed the same, good catch!